examples/cpp/dds/DynamicHelloWorldExample/HelloWorldPublisher.h

```diff
@@ -17,11 +17,10 @@
  *
  */
 
-#ifndef HELLOWORLDPUBLISHER_H_
-#define HELLOWORLDPUBLISHER_H_
+#ifndef FASTDDS_EXAMPLES_CPP_DDS_DYNAMIC_HELLO_WORLD_EXAMPLE__HELLO_WORLD_PUBLISHER_H
```

Comment:
Applies also for the old example header guards
```suggestion
#ifndef FASTDDS_EXAMPLES_CPP_DDS_DYNAMICHELLOWORLDEXAMPLE__HELLOWORLDPUBLISHER_H
```